libs/feature/search/src/lib/utils/service/aggregations.service.ts

```diff
@@ -0,0 +1,48 @@
+import { Injectable } from '@angular/core'
+import { SearchApiService } from '@geonetwork-ui/data-access/gn4'
+import { ElasticsearchService } from '@geonetwork-ui/util/shared'
+import { Observable } from 'rxjs'
+import { filter, map } from 'rxjs/operators'
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AggregationsService {
+  constructor(
+    private esService: ElasticsearchService,
+    private searchApiService: SearchApiService
+  ) {}
+
+  getAggregation(fieldName: string): Observable<any> {
+    return this.searchApiService
+      .search(
+        'bucket',
+        JSON.stringify(
+          this.esService.getSearchRequestBody(
```

Comment:
Seems the only added value of the method `esService.getSearchRequestBody` is to add the `template: 'n'` filter.
I think BTW that this should be refactored cause it shouldn't be hardcoded in this method.
Can be done for the editor search maybe